wandb/proto/wandb_internal.proto

```diff
@@ -122,6 +122,28 @@ message FooterRecord {
   _RecordInfo _info = 200;
 }
 
+// BranchPointRecord stores information about a branch point for a run.
+message BranchPointRecord {
+  // The name of the source run to branch off of.
+  string source_run = 1;
+
+  // The name of the metric to use to find a branch point.
+  //
+  // Currently, this is always `_step`.
+  string metric_name = 2;
+
+  // The value of the metric to branch at.
+  double metric_value = 3;
+
+  // The type of branch point.
+  enum BranchPointType {
+    BRANCH_POINT_FORK = 0;
```

Comment:
https://protobuf.dev/programming-guides/style/#enums

The standard is for the zero value of an enum to represent an unset/unknown value. This way, you don't need nil checks to access a nested enum field; instead `runRecord.GetBranchPoint().GetType()` will return the UNKNOWN value.

Let's fix this before the next release, so that it doesn't end up in any transaction logs!

Reply:
(I'll send a PR for it)